code/modules/wiremod/components/hud/click_intercept.dm

```diff
@@ -0,0 +1,64 @@
+/**
+ * # Click Intercept Component
+ *
+ * When activated intercepts next click and outputs clicked atom.
+ * Requires a BCI shell.
+ */
+
+/obj/item/circuit_component/click_intercept
+	display_name = "Click Intercept"
+	display_desc = "A component that when activated will intercept next user's click and output the object that was clicked. Requires a shell."
+
+	required_shells = list(/obj/item/organ/cyberimp/bci)
+
+	var/datum/port/output/clicked_atom
+
+	var/obj/item/organ/cyberimp/bci/bci
+	var/intercept_cooldown = 1 SECONDS
+
+/obj/item/circuit_component/click_intercept/Initialize()
+	. = ..()
+	trigger_input = add_input_port("Activate", PORT_TYPE_SIGNAL)
```

Comment:
There doesn't appear to be a way to deactivate this? Or at least, the name doesn't imply as such.

Reply:
You wait until user clicks on something